### ISSUE TITLE
[MStore Multivendor] - CeBuilder not recognizing expo installation,  Build via Metro Bundler hits error

### ISSUE THREAD

**Issue by selasegithub** [inspireui/support#659]
> - Thank you for reaching please follow the format below to create a new ticket, please explain as more detail as possible and allow 24 - 48 hours to receive an answer to your ticket.
> - After creating an issue, please copy the URL to the Product link on http://codecanyon.net - to help us know you have purchased our product (example: https://tppr.me/DJA8L)
> - Try to explain your all problems in one ticket, instead of creating multi tickets 

---------

1. My issue
I cant get CeBuilder to start the MStore Multivendor app. I have followed the installation guide and configured the app as directed. I installed all dependencies for the app via terminal as I'm running on a MAC. Terminal shows expo is installed as per the screenshots attached but when i Launch CeBuilder, the checks show expo is not installed. I could not proceed from that point so i tried to run the app via terminal using Metro Bundler but encountered the following error as shown in the screenshot.

I gave it another try and it still didn't work but rather displayed the following error codes.

Can't find variable: self

<unknown>
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:16636:46
<unknown>
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:16631:75
<unknown>
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:16632:5
loadModuleImplementation
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:159:14
<unknown>
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:16612:30
loadModuleImplementation
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:159:14
<unknown>
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:15544:21
getValue
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:3755:21
<unknown>
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:209459:13
<unknown>
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:209943:5
loadModuleImplementation
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:159:14
<unknown>
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:209437:11
loadModuleImplementation
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:159:14
<unknown>
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:209422:11
loadModuleImplementation
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:159:14
<unknown>
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:209377:27
loadModuleImplementation
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:159:14
<unknown>
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:198939:29
loadModuleImplementation
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:159:14
<unknown>
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:192688:27
loadModuleImplementation
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:159:14
<unknown>
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:192436:31
loadModuleImplementation
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:159:14
<unknown>
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:192357:25
loadModuleImplementation
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:159:14
<unknown>
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:124373:33
loadModuleImplementation
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:159:14
<unknown>
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:1426:22
loadModuleImplementation
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:159:14
guardedLoadModule
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:84:47
global code
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false&assetPlugin=%2FUsers%2Fkqwameselase%2FDownloads%2Fmstore-multi-vendor-v1.3.0%2Fmstore-mv-expo%2Fnode_modules%2Fexpo%2Ftools%2FhashAssetFiles.js:343318:8

2. Screenshots
<img width="1025" alt="Screen Shot 2019-05-30 at 19 49 35" src="https://user-images.githubusercontent.com/1086575/58680215-52bc2980-8356-11e9-88fa-666d44785efc.png">
<img width="1008" alt="Screen Shot 2019-05-30 at 19 49 44" src="https://user-images.githubusercontent.com/1086575/58680216-52bc2980-8356-11e9-8523-794485f92af6.png">
<img width="492" alt="Screen Shot 2019-05-30 at 19 50 37" src="https://user-images.githubusercontent.com/1086575/58680217-52bc2980-8356-11e9-8151-3ea85b62dc2e.png">
<img width="1432" alt="Screen Shot 2019-05-30 at 19 51 40" src="https://user-images.githubusercontent.com/1086575/58680218-5354c000-8356-11e9-97d6-ead6e33065de.png">
<img width="1427" alt="Screen Shot 2019-05-30 at 19 51 53" src="https://user-images.githubusercontent.com/1086575/58680220-5354c000-8356-11e9-843c-3873f0a4bb08.png">

3. Config files (optional)

**Comment by minhcasi** [comment]
Hi, this is caching issue, could you try to run this script from the project folder - 

`watchman watch-del-all && rm -rf $TMPDIR/react-* && rm -rf node_modules/ && yarn cache clean && yarn install && yarn start -- --reset-cache`

**Comment by selasegithub** [comment]
Thank you Minh,

 I actually did several times but CeBuilder still seems to find issues with expo not installed. I however can’t get started modifying the app using CeBuilder. See as per image below..
[cid:88E290DD-8266-4B07-9B58-6686A0E4A344]



Alternatively after running your code, all steps check out great except for the last part, “yarn start — —reset-cache”. There’s an issue with the — —reset-cache option throwing an error. See in the attachment below.
[cid:2302611B-E6A9-4015-8494-52BC207FE7B9]

So I had to run yarn start without the option and it build the app via Metro Bundler. However with that step as well there was an issue getting the simulator running from my end. I managed to run is on my iOS phone okay but testing out the registration feature raises an error.

Metro Bundler error screenshot

[cid:633B8D6D-E580-48CF-A8B6-B5B7D7A5A60E]


Error from app >>


Invariant Violation: Objects are not valid as a React child (found: SyntaxError: JSON Parse error: Unrecognized token '<'). If you meant to render a collection of children, use an array instead.
    in RCTText (at Text.js:202)
    in Text (at index.js:54)
    in RCTView (at View.js:60)
    in View (at createAnimatedComponent.js:154)
    in AnimatedComponent (at TouchableOpacity.js:247)
    in TouchableOpacity (at index.js:53)
    in RCTView (at View.js:60)
    in View (at index.js:45)
    in MyToast (created by Connect(MyToast))
    in Connect(MyToast) (at Router.js:65)
    in RCTView (at View.js:60)
    in View (at Router.js:59)
    in RCTView (at View.js:60)
    in View (at index.js:579)
    in RCTView (at View.js:60)
    in View (at index.js:566)
    in Drawer (at MenuOverlay.js:26)
    in MenuOverlay (created by Connect(MenuOverlay))
    in Connect(MenuOverlay) (at Router.js:56)
    in Router
    in ThemeProvider
    in withTheme(Router) (created by Connect(withTheme(Router)))
    in Connect(withTheme(Router)) (at App.js:124)
    in ThemeProvider (at Provider.js:17)
    in RCTView (at View.js:60)
    in View (at PortalHost.js:132)
    in Portal.Host (at Provider.js:16)
    in Provider (at App.js:123)
    in PersistGate (at App.js:122)
    in Provider (at App.js:121)
    in App (at registerRootComponent.js:35)
    in RootErrorBoundary (at registerRootComponent.js:34)
    in ExpoRootComponent (at renderApplication.js:33)
    in RCTView (at View.js:60)
    in View (at AppContainer.js:102)
    in RCTView (at View.js:60)
    in View (at AppContainer.js:122)
    in AppContainer (at renderApplication.js:32)

This error is located at:
    in RCTText (at Text.js:202)
    in Text (at index.js:54)
    in RCTView (at View.js:60)
    in View (at createAnimatedComponent.js:154)
    in AnimatedComponent (at TouchableOpacity.js:247)
    in TouchableOpacity (at index.js:53)
    in RCTView (at View.js:60)
    in View (at index.js:45)
    in MyToast (created by Connect(MyToast))
    in Connect(MyToast) (at Router.js:65)
    in RCTView (at View.js:60)
    in View (at Router.js:59)
    in RCTView (at View.js:60)
    in View (at index.js:579)
    in RCTView (at View.js:60)
    in View (at index.js:566)
    in Drawer (at MenuOverlay.js:26)
    in MenuOverlay (created by Connect(MenuOverlay))
    in Connect(MenuOverlay) (at Router.js:56)
    in Router
    in ThemeProvider
    in withTheme(Router) (created by Connect(withTheme(Router)))
    in Connect(withTheme(Router)) (at App.js:124)
    in ThemeProvider (at Provider.js:17)
    in RCTView (at View.js:60)
    in View (at PortalHost.js:132)
    in Portal.Host (at Provider.js:16)
    in Provider (at App.js:123)
    in PersistGate (at App.js:122)
    in Provider (at App.js:121)
    in App (at registerRootComponent.js:35)
    in RootErrorBoundary (at registerRootComponent.js:34)
    in ExpoRootComponent (at renderApplication.js:33)
    in RCTView (at View.js:60)
    in View (at AppContainer.js:102)
    in RCTView (at View.js:60)
    in View (at AppContainer.js:122)
    in AppContainer (at renderApplication.js:32)

throwOnInvalidObjectType
    ReactNativeRenderer-dev.js:7436:6
reconcileChildFibers
    ReactNativeRenderer-dev.js:8470:31
reconcileChildrenAtExpirationTime
    ReactNativeRenderer-dev.js:8622:8
reconcileChildren
    ReactNativeRenderer-dev.js:8603:6
updateHostComponent
    ReactNativeRenderer-dev.js:9001:22
performUnitOfWork
    ReactNativeRenderer-dev.js:12924:25
workLoop
    ReactNativeRenderer-dev.js:12953:43
renderRoot
    ReactNativeRenderer-dev.js:12996:17
performWorkOnRoot
    ReactNativeRenderer-dev.js:13632:34
performWork
    ReactNativeRenderer-dev.js:13545:26
performSyncWork
    ReactNativeRenderer-dev.js:13506:16
requestWork
    ReactNativeRenderer-dev.js:13392:6
scheduleWorkImpl
    ReactNativeRenderer-dev.js:13259:24
enqueueSetState
    ReactNativeRenderer-dev.js:6224:19
setState
    react.development.js:364:31
onStateChange
    connectAdvanced.js:220:24
onStateChange
    [native code]:0
notify
    Subscription.js:27:8
notifyNestedSubs
    Subscription.js:66:4
notifyNestedSubs
    [native code]:0
onStateChange
    connectAdvanced.js:217:10
onStateChange
    [native code]:0
notify
    Subscription.js:27:8
notifyNestedSubs
    Subscription.js:66:4
notifyNestedSubs
    [native code]:0
onStateChange
    connectAdvanced.js:217:10
onStateChange
    [native code]:0
dispatch
    redux.js:220:6
<unknown>
    index.js:1:3760
addToast
    index.js:97:37
doToast
    index.js:62:13
proxiedMethod
    [native code]:0
emit
    EventEmitter.js:180:12
toast
    Omni.js:105:20
stopAndToast
    index.js:145:10
_callee$
    index.js:103:31
tryCatch
    runtime.js:62:44
invoke
    runtime.js:296:30
tryCatch
    runtime.js:62:44
invoke
    runtime.js:152:28
<unknown>
    runtime.js:162:19
tryCallOne
    core.js:37:14
<unknown>
    core.js:123:25
_callTimer
    JSTimers.js:152:6
_callImmediatesPass
    JSTimers.js:200:17
callImmediates
    JSTimers.js:464:11
__callImmediates
    MessageQueue.js:327:4
<unknown>
    MessageQueue.js:145:6
__guardSafe
    MessageQueue.js:314:6
flushedQueue
    MessageQueue.js:144:17
flushedQueue
    [native code]:0
invokeCallbackAndReturnFlushedQueue
    [native code]:0


Its been an extremely frustrating week just to set up and get running.

I thought I was choosing the fastest simplest way out but looks like hell right now and I will appreciate your help greatly,

Thanks.

On Jun 1, 2019, at 11:17, Minh Pham <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi, this is caching issue, could you try to run this script from the project folder -

watchman watch-del-all && rm -rf $TMPDIR/react-* && rm -rf node_modules/ && yarn cache clean && yarn install && yarn start -- --reset-cache

**Comment by selasegithub** [comment]
I tried one more time with npm and with yarn but neither was successful with reseting the cache. Snapshot below.

[cid:804AFBDB-901D-4226-9251-23D9717ACCE4]

On Jun 1, 2019, at 14:20, Richard Kwame Selase Addrah <proudryan@hotmail.com<mailto:proudryan@hotmail.com>> wrote:

Thank you Minh,

 I actually did several times but CeBuilder still seems to find issues with expo not installed. I however can’t get started modifying the app using CeBuilder. See as per image below..
<Screen Shot 2019-06-01 at 14.14.08.png>



Alternatively after running your code, all steps check out great except for the last part, “yarn start — —reset-cache”. There’s an issue with the — —reset-cache option throwing an error. See in the attachment below.
<Screen Shot 2019-06-01 at 14.11.07.png>

So I had to run yarn start without the option and it build the app via Metro Bundler. However with that step as well there was an issue getting the simulator running from my end. I managed to run is on my iOS phone okay but testing out the registration feature raises an error.

Metro Bundler error screenshot

<Screen Shot 2019-06-01 at 14.08.04.png>


Error from app >>


Invariant Violation: Objects are not valid as a React child (found: SyntaxError: JSON Parse error: Unrecognized token '<'). If you meant to render a collection of children, use an array instead.
    in RCTText (at Text.js:202)
    in Text (at index.js:54)
    in RCTView (at View.js:60)
    in View (at createAnimatedComponent.js:154)
    in AnimatedComponent (at TouchableOpacity.js:247)
    in TouchableOpacity (at index.js:53)
    in RCTView (at View.js:60)
    in View (at index.js:45)
    in MyToast (created by Connect(MyToast))
    in Connect(MyToast) (at Router.js:65)
    in RCTView (at View.js:60)
    in View (at Router.js:59)
    in RCTView (at View.js:60)
    in View (at index.js:579)
    in RCTView (at View.js:60)
    in View (at index.js:566)
    in Drawer (at MenuOverlay.js:26)
    in MenuOverlay (created by Connect(MenuOverlay))
    in Connect(MenuOverlay) (at Router.js:56)
    in Router
    in ThemeProvider
    in withTheme(Router) (created by Connect(withTheme(Router)))
    in Connect(withTheme(Router)) (at App.js:124)
    in ThemeProvider (at Provider.js:17)
    in RCTView (at View.js:60)
    in View (at PortalHost.js:132)
    in Portal.Host (at Provider.js:16)
    in Provider (at App.js:123)
    in PersistGate (at App.js:122)
    in Provider (at App.js:121)
    in App (at registerRootComponent.js:35)
    in RootErrorBoundary (at registerRootComponent.js:34)
    in ExpoRootComponent (at renderApplication.js:33)
    in RCTView (at View.js:60)
    in View (at AppContainer.js:102)
    in RCTView (at View.js:60)
    in View (at AppContainer.js:122)
    in AppContainer (at renderApplication.js:32)

This error is located at:
    in RCTText (at Text.js:202)
    in Text (at index.js:54)
    in RCTView (at View.js:60)
    in View (at createAnimatedComponent.js:154)
    in AnimatedComponent (at TouchableOpacity.js:247)
    in TouchableOpacity (at index.js:53)
    in RCTView (at View.js:60)
    in View (at index.js:45)
    in MyToast (created by Connect(MyToast))
    in Connect(MyToast) (at Router.js:65)
    in RCTView (at View.js:60)
    in View (at Router.js:59)
    in RCTView (at View.js:60)
    in View (at index.js:579)
    in RCTView (at View.js:60)
    in View (at index.js:566)
    in Drawer (at MenuOverlay.js:26)
    in MenuOverlay (created by Connect(MenuOverlay))
    in Connect(MenuOverlay) (at Router.js:56)
    in Router
    in ThemeProvider
    in withTheme(Router) (created by Connect(withTheme(Router)))
    in Connect(withTheme(Router)) (at App.js:124)
    in ThemeProvider (at Provider.js:17)
    in RCTView (at View.js:60)
    in View (at PortalHost.js:132)
    in Portal.Host (at Provider.js:16)
    in Provider (at App.js:123)
    in PersistGate (at App.js:122)
    in Provider (at App.js:121)
    in App (at registerRootComponent.js:35)
    in RootErrorBoundary (at registerRootComponent.js:34)
    in ExpoRootComponent (at renderApplication.js:33)
    in RCTView (at View.js:60)
    in View (at AppContainer.js:102)
    in RCTView (at View.js:60)
    in View (at AppContainer.js:122)
    in AppContainer (at renderApplication.js:32)

throwOnInvalidObjectType
    ReactNativeRenderer-dev.js:7436:6
reconcileChildFibers
    ReactNativeRenderer-dev.js:8470:31
reconcileChildrenAtExpirationTime
    ReactNativeRenderer-dev.js:8622:8
reconcileChildren
    ReactNativeRenderer-dev.js:8603:6
updateHostComponent
    ReactNativeRenderer-dev.js:9001:22
performUnitOfWork
    ReactNativeRenderer-dev.js:12924:25
workLoop
    ReactNativeRenderer-dev.js:12953:43
renderRoot
    ReactNativeRenderer-dev.js:12996:17
performWorkOnRoot
    ReactNativeRenderer-dev.js:13632:34
performWork
    ReactNativeRenderer-dev.js:13545:26
performSyncWork
    ReactNativeRenderer-dev.js:13506:16
requestWork
    ReactNativeRenderer-dev.js:13392:6
scheduleWorkImpl
    ReactNativeRenderer-dev.js:13259:24
enqueueSetState
    ReactNativeRenderer-dev.js:6224:19
setState
    react.development.js:364:31
onStateChange
    connectAdvanced.js:220:24
onStateChange
    [native code]:0
notify
    Subscription.js:27:8
notifyNestedSubs
    Subscription.js:66:4
notifyNestedSubs
    [native code]:0
onStateChange
    connectAdvanced.js:217:10
onStateChange
    [native code]:0
notify
    Subscription.js:27:8
notifyNestedSubs
    Subscription.js:66:4
notifyNestedSubs
    [native code]:0
onStateChange
    connectAdvanced.js:217:10
onStateChange
    [native code]:0
dispatch
    redux.js:220:6
<unknown>
    index.js:1:3760
addToast
    index.js:97:37
doToast
    index.js:62:13
proxiedMethod
    [native code]:0
emit
    EventEmitter.js:180:12
toast
    Omni.js:105:20
stopAndToast
    index.js:145:10
_callee$
    index.js:103:31
tryCatch
    runtime.js:62:44
invoke
    runtime.js:296:30
tryCatch
    runtime.js:62:44
invoke
    runtime.js:152:28
<unknown>
    runtime.js:162:19
tryCallOne
    core.js:37:14
<unknown>
    core.js:123:25
_callTimer
    JSTimers.js:152:6
_callImmediatesPass
    JSTimers.js:200:17
callImmediates
    JSTimers.js:464:11
__callImmediates
    MessageQueue.js:327:4
<unknown>
    MessageQueue.js:145:6
__guardSafe
    MessageQueue.js:314:6
flushedQueue
    MessageQueue.js:144:17
flushedQueue
    [native code]:0
invokeCallbackAndReturnFlushedQueue
    [native code]:0


Its been an extremely frustrating week just to set up and get running.

I thought I was choosing the fastest simplest way out but looks like hell right now and I will appreciate your help greatly,

Thanks.

On Jun 1, 2019, at 11:17, Minh Pham <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi, this is caching issue, could you try to run this script from the project folder -

watchman watch-del-all && rm -rf $TMPDIR/react-* && rm -rf node_modules/ && yarn cache clean && yarn install && yarn start -- --reset-cache

**Comment by selasegithub** [comment]
This has been quite frustrating but i managed to get a fix no thanks to support Team. I did many things and not quite sure what works but off the top of my head, I installed expo and cleared the cache. I also reinstalled CeBuilder. 
In the end, i had to eject expo as it was very limiting with the customizations i had to make. All the same thanks and i hope support can improve greatly to respond faster and be more helpful. 

Thank you.
